pkg/tags/tag.go

```diff
@@ -280,3 +300,21 @@ func decodeInt64Splice(buffer *[]byte) int64 {
 	*buffer = (*buffer)[n:]
 	return val
 }
+
+// UpdateTag update the tag in the state store
+func (tag *Tag) UpdateTag() {
```

Comment:
UpdateTag probably does not need to be exported as it is not used outside the package

Reply:
I agree, and a name like `persistTag` or `saveTag` would be more appropriate.